qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/ls/api/QuteProjectInfoProvider.java

```diff
@@ -26,6 +28,38 @@
  */
 public interface QuteProjectInfoProvider {
 
+	/**
+	 * Returns all Qute projects from the workspace.
+	 * 
+	 * @return all Qute projects from the workspace.
+	 */
+	@JsonRequest("qute/template/projects")

```

Comment:
why "qute/template/projects" instead of "qute/projects"? Project changes will affect both template and java support

Reply:
I did that because all qute/template/* service are managed with `QuteSupportForTemplate`. More  "qute/template/projects" are used by only Qute template support. Java files are managed by Java external component IJ.Eclipseide